frontend/src/lib/components/modals/first-time-setup/FirstTimeSetupModal.svelte

```diff
@@ -0,0 +1,115 @@
+<script lang="ts">
+  import { mdiTimerSandFull, mdiWeb } from '@mdi/js';
+  import { SlideToggle } from '@skeletonlabs/skeleton';
+  import { T } from '@tolgee/svelte';
+
+  import LanguageSelector from './LanguageSelector.svelte';
+
+  import SvgIcon from '$lib/components/SVGIcon.svelte';
+  import { queueAutoStart } from '$lib/store/settingsStore';
+  import { SetNewUserSetupComplete } from '$wailsjs/go/settings/settings';
+  import { BrowserOpenURL } from '$wailsjs/runtime/runtime';
+
+  export let parent: { onClose: () => void };
+
+  const OpenWelcomeGuide = () => {
+    BrowserOpenURL('https://docs.ficsit.app/satisfactory-modding/latest/ForUsers/Welcome.html');
+  };
+
+  function onClose() {
+    SetNewUserSetupComplete(true);
+    parent.onClose();
+  }
+</script>
+
+<div
+  style="max-height: calc(100vh - 3rem); max-width: calc(100vw - 3rem);"
+  class="w-[48rem] card flex flex-col gap-6"
+>
+  <header class="card-header font-bold text-2xl text-center">
+    <T defaultValue="Welcome to the Satisfactory Mod Manager!" keyName="first_time_setup.title" />
+  </header>
+  <section class="px-4">
+    <p class="text-base text-center">
+      <T
+        defaultValue="Select your preferences to get started."
+        keyName="first_time_setup.intro"
+      />
+    </p>
+  </section>
+  <section class="px-4 overflow-y-auto">
```

Comment:
`overflow-y-auto` limits the space for the select popup to this section
```suggestion
  <section class="px-4 overflow-y-visible">
```

Reply:
Thank you, I burned a frustrating amount of time trying to figure this out